### PR TITLE
Refresh understanding-guest-agents-and-other-tools

### PR DESCRIPTION
**Pull request by sandrobonazzola**
Changes proposed in this pull request:

- Refresh "Understanding Guest Agents and other tools"

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): @sandrobonazzola 

This pull request needs review by: @ahadas , @nyoxi , @Gal-Zaidman

### CODE REVIEW

File: source/develop/internal/guest-agent/understanding-guest-agents-and-other-tools.md [hunk 1]
```diff
@@ -1,25 +1,40 @@
...
 
-The oVirt Guest Agent provides information, notifications, and actions between the oVirt web interface and the guest. A more detailed description can be found at [Guest Agent](guest-agent). The agent provides the Machine Name, Operating System, IP Addresses, Installed Applications, Network and RAM usage and others details to the web interface. The agent also provides Single Sign On so a authenticated user to the web interface does not need to authenticate again when connected to a VM.
+**The oVirt Guest Agent has been deprecated** in favor of QEMU Guest Agent and is not available anymore for recent OS. It can still be found for less recent OS in their own distribution repositories.
+The oVirt Guest Agent provides information, notifications, and actions between the oVirt web interface and the guest. A more detailed description can be found at [oVirt Guest Agent](guest-agent.html).
+The agent provides the Machine Name, Operating System, IP Addresses, Installed Applications, Network and RAM usage and others details to the web interface. The agent also provides Single Sign On 
+so a authenticated user to the web interface does not need to authenticate again when connected to a VM.
```
Comment (ahadas) [inline source/develop/internal/guest-agent/understanding-guest-agents-and-other-tools.md:31]: so an authentication user*

File: source/develop/internal/guest-agent/understanding-guest-agents-and-other-tools.md [hunk 2]
```diff
@@ -1,25 +1,40 @@
...
+
+### QEMU Guest Agent
+
+The QEMU Guest Agent helps management applications with executing functions which need assistance of the guest OS. For example, freezing and thawing filesystems, entering suspend.
+It provides information, notifications, and actions between the oVirt web interface and the guest. A more detailed description can be found at [QEMU Guest Agent](https://wiki.libvirt.org/page/Qemu_guest_agent).
```
Comment (ahadas) [inline source/develop/internal/guest-agent/understanding-guest-agents-and-other-tools.md:36]: maybe worth putting it before the oVirt Guest Agent?

File: source/develop/internal/guest-agent/understanding-guest-agents-and-other-tools.md (hunk 1 above)
Comment (sandrobonazzola) [reply source/develop/internal/guest-agent/understanding-guest-agents-and-other-tools.md:31]: fixing

File: source/develop/internal/guest-agent/understanding-guest-agents-and-other-tools.md (hunk 2 above)
Comment (sandrobonazzola) [reply source/develop/internal/guest-agent/understanding-guest-agents-and-other-tools.md:36]: moved

**Pull request merged**